metaflow/plugins/kfp/kfp.py

```diff
@@ -84,6 +85,7 @@ def __init__(
         self.workflow_timeout = (
             workflow_timeout if workflow_timeout else 0  # 0 is unlimited
         )
+        self.workflow_ttl = workflow_ttl if workflow_ttl else 604800  # 1 week
```

Comment:
`workflow_ttl` -> `workflow_kubernetes_resources_ttl`

Reply:
Is there an Argo TTL setting that we can set in it's manifest or configuration instead of it being client side?

Reply:
Argo Garbage Collector concerns and settings ideally should be handled by Argo and it's configuration rather than all clients (KFP and Metaflow)